docs/docs/guides/how-to-seed-mysql.md

```diff
@@ -0,0 +1,209 @@
+
+```python
+. . .
+   return {
+       service = mysql_service,
+       name = database_name,
+       user = database_user,
+       password = database_password,
+   }
+. . .
+```
+:::note
+Note that we made this function generic enough so any team can leverage this package, independent of the application, by passing the desired database name, credentials, and setup script, abstracting away MySQL specifics and only interacting with the result of create_database.
+:::
+
+You now have all the instructions needed for Kurtosis to instantiate a MySQL database. However, a databse is not very useful unless its queryable by other users. For consumers of this package to run a SQL query after database creation, add the following block that uses the result of `create_database` as an input:
+
+```python
+def run_sql(plan, database, sql_query):
+   exec_result = plan.exec(
+       service_name = database.service.name,
+   )
+   return exec_result["output"]
+```
+And thats it! What you've done here is write a set of instructions that anyone can consume and use, with Kurtosis, to instantiate a MySQL database and make a simple query against it to test functionality. Your local `mysql.star` should look identical to this one on [Github](https://github.com/kurtosis-tech/mysql-package/blob/main/mysql.star)
+
+Seed the database with data upon startup
+----------------------------------------
+With the MySQL package ready to go, you will now write a new package that seeds it with data. In this specific case, you will write a package that spins up a database pre-populated with blog data to test our application.
+
+Start by creating a folder called "blog-mysql-seed" next to your "mysql-package" folder. Inside your "blog-mysql-seed" folder, create a `blog-mysql-seed/kurtosis.yml` file with the following contents:
+```yml
+name: github.com/kurtosis-tech/awesome-kurtosis/blog-mysql-seed
+```
+
+Next, create 2 `.sql` files: "setup.sql" to create tables and "seed.sql" to populate them, and save them in the same folder.
+```sql
+% setup.sql
+CREATE TABLE user (
+   user_id INT NOT NULL,
+   first_name VARCHAR(255) NOT NULL,
+   PRIMARY KEY (user_id)
+);
+CREATE TABLE post (
+   post_id INT NOT NULL AUTO_INCREMENT,
+   content VARCHAR(255) NOT NULL,
+   author_user_id INT NOT NULL,
+   PRIMARY KEY (post_id),
+   FOREIGN KEY (author_user_id) REFERENCES user(user_id)
+);
+
+% seed.sql
+INSERT INTO user (user_id, first_name) VALUES (1, "Peter");
+INSERT INTO user (user_id, first_name) VALUES (2, "Jorge");
+INSERT INTO post (post_id, content, author_user_id) VALUES (0, "Lorem ipsum dolor sit amet", 2);
+```
+
+Finally, create a `main.star` file with the following contents:
+
+```python
+mysql = import_module("github.com/kurtosis-tech/mysql-package/mysql.star")
+
+SELECT_SQL_EXAMPLE = "SELECT * FROM post"
+
+def run(plan, args):
+    setup_sql = plan.upload_files(
+        src = "github.com/kurtosis-tech/awesome-kurtosis/blog-mysql-seed/setup.sql",
```

Comment:
@victorcolombo is there a way we can upload_files and read_file from the locally created .sql files rather than from Github? Otherwise this feels like a vulnerability and also defeats the purpose of having a user write the .sql files locally to begin with.